src/main/java/com/example/graduationproject/Controller/UsersModelController.java

```diff
@@ -11,6 +11,7 @@
 
 @Controller
 public class UsersModelController {
+    //
```

Comment:
model.addAttribute("registerRequest" change to model.addAttribute("user"

Reply:
done